plugins/plugin_utils.c

```diff
@@ -97,6 +98,19 @@ int32_t deserialize_meta(uint8_t *smeta, int32_t smeta_len, int8_t *ndim, int64_
   return slen;
 }
 
+void index_unidim_to_multidim(uint8_t ndim, int64_t *shape, int64_t i, int64_t *index) {
+    int64_t strides[8];
```

Comment:
This 8 could be a constant, like `BLOSC2_MAXDIM` in blosc2.h.

Reply:
no m'apareix cap constant BLOSC2_MAXDIM

Reply:
Yeah, this is why I am suggesting to *add* it :-)